core/src/main/scala/org/http4s/internal/parsing/Rfc7230.scala

```diff
@@ -57,11 +57,26 @@ private[http4s] object Rfc7230 {
 
   private def surroundedBy[A](a: Parser0[A], b: Parser[Any]): Parser[A] =
     b *> a <* b
+  private def between[A](a: Parser[Any], b: Parser0[A], c: Parser[Any]): Parser[A] =
+    a *> b <* c
 
   /*quoted-string  = DQUOTE *( qdtext / quoted-pair ) DQUOTE*/
   val quotedString: Parser[String] =
     surroundedBy(qdText.orElse(quotedPair).rep0.string, dquote)
 
+  /* HTAB / SP / %x21-27 / %x2A-5B / %x5D-7E / obs-text */
+  val CText =
```

Comment:
Nit: all the other parsers are lowercase names.

Reply:
https://github.com/http4s/http4s/pull/4208/commits/c6a368b9b9a64f5744880ec3628ed806e93e3f5a